### PR TITLE
Fix design spec

### PR DESCRIPTION
**Pull request by FacundoPucheta**
- New helper 'asce' created
- Added new type 'asce' into designSpectrum controller AND handle the validation.
- New tests created for request wit 'type=asce' as query. ( /designspectrum  route)

### CODE REVIEW

File: src/helpers/asce.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  return F[tp][5];
+};
+
+const asce = async (location, soilType) => {
+
+  let prob = await probabilities(location, 0.2);
+  let Ss = interpolation(prob, 1.0 / 2475.0); //PGA1
+
+  let prob2 = await probabilities(location, 1.0);
+  let Sl = interpolation(prob2, 1.0 / 2500.0); //PGA2
```
Comment (SebaSatelier) [inline src/helpers/asce.js]: this line is  let Sl = interpolation(prob2, 1.0 / 2475.0) or i'm wrong?

File: src/helpers/asce.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  if (Sl < 0.08) Sl = 0.08;
```
Comment (SebaSatelier) [inline src/helpers/asce.js]: In PHP code, this line is comment, not in use

Comment (FacundoPucheta) [reply src/helpers/asce.js:49]: Fixed ✔️

**Review by SebaSatelier — CHANGES_REQUESTED** [review]
I leave some comments

**Review by SebaSatelier — APPROVED** [review]
NIce job!

**Review by manduinca — APPROVED** [review]
LGTM!

**Pull request merged**